ext/imap/php_imap.c

```diff
@@ -417,6 +451,19 @@ PHP_MINIT_FUNCTION(imap)
 	ssl_onceonlyinit ();
 #endif
 
+
+	INIT_CLASS_ENTRY(ce, "IMAPConnection", NULL);
```

Comment:
Define your class in the .stub file and reference it's methods here as `class_IMAPConnection_methods`.

Since the class has no methods this distinction is minor, but we should be consistent about presenting user-facing APIs in the stub files, and once we have it in the stub file, we'll want the method list referenced to avoid surprises for future-us.

Reply:
Done :)